optimism/deployments/utils/dependencies.ts

```diff
@@ -26,11 +26,18 @@ export default {
     },
     69: "0x6027c4dcFE97fa079D639bBDD018E95dDc1d9c65"
   },
+  TEST_BASIS_TOKEN: {
+    10: {
+      staging: "0x337Ee7DD69bE412763306e167A155c97F0fBE645",
+      production: ""
+    },
+    69: "0xfA60d10BD76E4Ad2d4A88271f6cd0CA88eDBd08f"
+  },
 
   // PerpV2 virtual tokens
   V_ETH: {
     10: "0x8C835DFaA34e2AE61775e80EE29E2c724c6AE2BB",
-    69: "0x7cE531e940B24EE760e5dDBF8A2E0E359fA04400"
+    69: "0x099FaDE6D13401c4f9742db09ae827E434FB4C30"
```

Comment:
They changed this? (I see it's valid - `vETH2` - just wondering if there's a reason why)

Reply:
0x8C835DFaA34e2AE61775e80EE29E2c724c6AE2BB looks ok any - getting hit